src/planscape/planning/tests.py

```diff
@@ -363,3 +363,383 @@ def test_end_to_end(self):
             content_type="application/json")
         self.assertEqual(response.status_code, 200)
         self.assertEqual(len(response.json()), 0)
+
+
+#### SCENARIO Tests ####
+
+# Blindly create a scenario and a scenario result in its default (pending) state.
+# Note that this does no deduplication, which our APIs may eventually do.
+def _create_scenario(plan: PlanningArea, scenario_name: str, configuration: str) -> Scenario:
+    scenario = Scenario.objects.create(
+        planning_area=plan, name=scenario_name, configuration=configuration)
+    scenario.save()
+
+    scenario_result = ScenarioResult.objects.create(
+        scenario=scenario)
+    scenario_result.save()
+
+    return scenario
+
+
+class CreateScenarioTest(TransactionTestCase):
+    def setUp(self):
+        self.user = User.objects.create(username='testuser')
+        self.user.set_password('12345')
+        self.user.save()
+
+        self.geometry = {'type': 'MultiPolygon',
+                         'coordinates': [[[[1, 2], [2, 3], [3, 4], [1, 2]]]]}
+        self.stored_geometry = GEOSGeometry(json.dumps(self.geometry))
+        self.plan = _create_plan(self.user, 'test plan', self.stored_geometry)
+
+        self.user2 = User.objects.create(username='testuser2')
+        self.user2.set_password('12345')
+        self.user2.save()
+        self.plan2 = _create_plan(self.user2, 'test plan 2', self.stored_geometry)
+        
+    def test_create_scenario(self):
+        self.client.force_login(self.user)
+        response = self.client.post(
+            reverse('planning:create_scenario'),
+            {'plan_id': self.plan.pk,
+             'configuration': '{}',
+             'name': 'test scenario'},
+            content_type="application/json")
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(Scenario.objects.count(), 1)
+        self.assertEqual(ScenarioResult.objects.count(), 1)
+
+    def test_create_scenario_not_logged_in(self):
+        response = self.client.post(
+            reverse('planning:create_scenario'),
+            {'plan_id': self.plan.pk,
+             'configuration': '{}',
```

Comment:
Once we have configuration validation we'll want a test_configuration dictionary with basic values to pass in here. Might be worth adding a todo or making an empty one for now to pass in just as a reminder

Reply:
Yup - the TODO's on line 441.  I do have the empty config here in the tests, but if you already have some sort of structure to suggest, I'll use it here.

Reply:
So far it looks like an example one might look like:
{
    'est_cost': 0,
    'max_budget': 0,
    'max_road_distance', 0,
    'max_slope': 0,
    'priorities': ['priority1'],
    'weights': [0],
    'stand_size': 'Large',
   'excluded_areas': []
 }

Note for future: I believe 'max_budget' will be an optional config value